src/components/Battle.js

```diff
@@ -70,6 +70,7 @@ class Battle extends Component {
                     id={monster.monster.id}
                     hp={monster.monster.hp}
                     maxHp={monster.ref.hp}
+                    monsterMoves={this.props.allState.battle.monsterMoves}
```

Comment:
wouldn't it be better to just sent the monsterMoves for the specific monster in question here:

```suggestion
                    monsterMoves={this.props.allState.battle.monsterMoves[monster.monster.uuid]}
```